From 4e0a238ca281ffa94d41d400cb8518c49089d266 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Tue, 20 Aug 2013 16:40:29 +0800
Subject: [PATCH 5/6] ENGR00275370 Gallery2 edit SHARPNESS pops up error

com.android.gallry3d error pops up when select SHARPNESS from edit

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 src/com/android/gallery3d/filtershow/FilterShowActivity.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/com/android/gallery3d/filtershow/FilterShowActivity.java b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
index e627a61..e5899ad 100644
--- a/src/com/android/gallery3d/filtershow/FilterShowActivity.java
+++ b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
@@ -686,6 +686,8 @@ public class FilterShowActivity extends FragmentActivity implements OnItemClickL
             mCurrentEditor.detach();
         }
         mCurrentEditor = mEditorPlaceHolder.showEditor(representation.getEditorId());
+        if (mCurrentEditor == null)
+            mCurrentEditor = mEditorPlaceHolder.showEditor(EditorTinyPlanet.ID);
         loadEditorPanel(representation, mCurrentEditor);
     }
 
-- 
1.8.0

